<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#panel, #MainWindow{
	background-color:#13CE66;
}
#pb_inicio, #pb_pacientes, #pb_secretarios, #pb_medicos{
	font-size: 18px;
	background-color:#1B6C3F;
	border:none;
	color: white;
	text-align: center;
	height:60px;
    width: 120px;
}


</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="panel" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="topMargin">
          <number>100</number>
         </property>
         <property name="bottomMargin">
          <number>200</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="4" column="0">
          <widget class="QPushButton" name="pb_medicos">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Medicos</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="pb_pacientes">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Pacientes</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pb_inicio">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Inicio</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pb_secretarios">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Secretarios</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QMdiArea" name="mdiArea">
         <property name="styleSheet">
          <string notr="true">QMdiArea{
background: white;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pb_inicio</tabstop>
  <tabstop>pb_secretarios</tabstop>
  <tabstop>pb_pacientes</tabstop>
  <tabstop>pb_medicos</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
